classes/classes/PlayerAppearance.as

```diff
@@ -3,10 +3,46 @@ package classes
 	import classes.*; 
 	import classes.GlobalFlags.*;
 	
-	public class PlayerAppearance extends BaseContent
-	{
+	public class PlayerAppearance extends BaseContent {
+
 		public function PlayerAppearance() {}
 		
+		/*** Internal methods; for outputting text in proper measure system ***/
+
+		private function feet_inch_and_metres (in_inches:Number):String {
+			if (flags[kFLAGS.USE_METRICS])  return "" + (Math.round(in_inches * 2.54) / 100).toFixed(2) + " metres";
+			else                            return "" + Math.floor(in_inches / 12) + " foot " + in_inches % 12 + " inch";
+		}
+
+		private function num_inches_and_centimetres (in_inches:Number):String {
+			// Various special cases.
+			if (in_inches < 1)
+				return inches_and_centimetres(in_inches)
+			else if (in_inches == 12 && !flags[kFLAGS.USE_METRICS])
+				return "a foot";
+			else if (in_inches % 12 == 0 && !flags[kFLAGS.USE_METRICS])
+				return num2Text(in_inches / 12) + " feet";
+
+			// General
+			var value:int = int(in_inches * (flags[kFLAGS.USE_METRICS] ? 2.54 : 1));
+			if (flags[kFLAGS.USE_METRICS])  return "" + num2Text(value) + " centimetre" + (value == 1 ? "" : "s");
+			else                            return "" + num2Text(value) + " inch" +       (value == 1 ? "" : "es");
+		}
+
+		private function inches_and_centimetres (in_inches:Number, precision:int = 10):String {
+			var value:Number = Math.round(in_inches*precision * (flags[kFLAGS.USE_METRICS] ? 2.54 : 1)) / precision;
```

Comment:
personally I'd use `, precision:int = 1)` since precision usually refers to the number of digits after the decimal seperator and then use Math.pow(10, precision)
Although not applicable here additionally precision could be used in `.toFixed(precision)`